src/main/java/com/ystrazhko/git/connections/RESTConnector.java

```diff
@@ -5,5 +5,5 @@
 public interface RESTConnector {
     String URL_MAIN_PART = "https://gitlab.com/api/v3";
 
-    Object sendPost(String url, Map<String, String> params);
+    Object sendPost(String url, Map<String, String> params, Map<String, String> header, String reguest);

```

Comment:
I think we should add sendGet method to the interface and delegate sendPost and sendGet to this implementation